### PR TITLE
Documentation: Add contributing note for disabling actions in forks

### PR DESCRIPTION
**Pull request by johnhunter**
## What I did

- Added a note about disabling actions when forking the repo. Most forks will be used to dev locally on the project and raise PRs so the actions don't make sense and will just cause errors.
- Added an issues section to help contributors solve known issues.
- Fixed a broken link to the contributing guide

## How to test

Documentation only.
If required you can confirm contributing guide link does not 404.

## Checklist

- [ ] Make sure your changes are tested (stories and/or unit, integration, or end-to-end tests)
- [x] Make sure to add/update documentation regarding your changes
- [ ] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

- [ ] When this PR is ready for testing, make sure to add `ci:normal`, `ci:merged` or `ci:daily` GH label to it to run a specific set of sandboxes. The particular set of sandboxes can be found in `code/lib/cli/src/sandbox-templates.ts`
- [ ] Make sure this PR contains **one** of the labels below.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "build", "documentation", "maintenance", "dependencies", "other"]`

### 🦋 Canary release

This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_

### CODE REVIEW

**Comment by johnhunter** [comment]
Ok, failing on Danger JS 😳 - I made the docs changes in the GH site. I'll pull down the PR locally and have a look in a bit.

**Comment by johnhunter** [comment]
Am getting errors running `yarn start` locally so moving this PR to draft until resolved.

**Comment by johnhunter** [comment]
Ok, the failing CI on this PR is unrelated to my local issues so setting back to review.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,6 +7,17 @@ Storybook is developed against a specific node version. We recommend using [Volt
 
 The `yarn start` script will generate a React Vite TypeScript sandbox with a set of test stories inside it, as well as taking all steps required to get it running (building the various packages we need etc). There is no need to run `yarn` or `yarn install` as `yarn start` will do this for you.
 
+## Issues
+
+On `yarn start` you encounter the following error, try rerunning `yarn start` a second time:
```
Comment (kylegach) [inline CONTRIBUTING.md:12]:
```suggestion
If you run `yarn start` and encounter the following error, try rerunning `yarn start` a second time:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ Storybook is developed against a specific node version. We recommend using [Volt
...
+```
+>  NX   ENOENT: no such file or directory, open 'storybook/code/node_modules/nx/package.json'
+```
```
Comment (kylegach) [inline CONTRIBUTING.md:15]:
```suggestion
```sh
>  NX   ENOENT: no such file or directory, open 'storybook/code/node_modules/nx/package.json'
```
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -7,6 +7,17 @@ Storybook is developed against a specific node version. We recommend using [Volt
...
+
+## Forked repos
+
+If you have forked the repo you should disable the Github actions to avoid failing runs (e.g. pushing to sandbox). In your Github repo go to **settings** > **actions** and set the **Disable actions** setting.
```
Comment (kylegach) [inline CONTRIBUTING.md:19]:
```suggestion
If you have forked the repository, you should [disable Github Actions for your repo](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository) as many of them (e.g. pushing to sandbox) will fail without proper authorization. In your Github repo, go to Settings > Actions > General > set the Actions Permissions to **Disable actions**.
```

**Review by kylegach — COMMENTED** [review]
Thanks for these improvements, @johnhunter!

I left some suggested further improvements for you to integrate, then we can merge.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -7,6 +7,16 @@ Storybook is developed against a specific node version. We recommend using [Volt
 
 The `yarn start` script will generate a React Vite TypeScript sandbox with a set of test stories inside it, as well as taking all steps required to get it running (building the various packages we need etc). There is no need to run `yarn` or `yarn install` as `yarn start` will do this for you.
 
+## Issues
+
+If you run `yarn start` and encounter the following error, try rerunning `yarn start` a second time:
+```sh
+>  NX   ENOENT: no such file or directory, open 'storybook/code/node_modules/nx/package.json'
```
Comment (kylegach) [inline CONTRIBUTING.md:14]: Looks like something happened when addressing feedback or resolving merge conflicts?

````suggestion
If you run `yarn start` and encounter the following error, try rerunning `yarn start` a second time:

```sh
>  NX   ENOENT: no such file or directory, open 'storybook/code/node_modules/nx/package.json'
```
````

Comment (johnhunter) [reply CONTRIBUTING.md:14]: Ok. I'll have look when I'm back at my desk. Thanks.

**Review by kylegach — APPROVED** [review]
Nicely done! Thanks again, @johnhunter.

**Pull request merged**